python/dronin/telemetry.py

```diff
@@ -193,23 +197,70 @@ def __handle_handshake(self, obj):
 
             if obj.Status == obj.ENUM_Status['Disconnected']:
                 # Request handshake
-                print("Disconnected")
+                #print("Disconnected")
```

Comment:
Yah, these were problematic-- corrupted my received files.  Ditto for the bad CRC messages.